k8s/test/e2e/failover/e2e.yaml

```diff
@@ -68,29 +68,46 @@ setup:
     - name: install backup
       command: |
         kubectl create ns backup
-        kubectl apply -f k8s/test/e2e/failover-demo/backup.yaml
-        while ! [[ $(kubectl get backup backup-sample -n backup -o jsonpath='{.status.state}') = 'Succeed'  ]]; \
-        do \
-          sleep 0.1; \
-        done
+        go run k8s/cmd/main.go -n backup create backup \
+        --vineyardd-name vineyardd-sample \
+        --vineyardd-namespace vineyard-system  \
+        --limit 1000 \
+        --path /var/vineyard/dump  \
+        --pv-pvc-spec '{
+        "pv-spec": {
+          "capacity": {
+            "storage": "1Gi"
+          },
+          "accessModes": [
+            "ReadWriteOnce"
+          ],
+          "storageClassName": "manual",
+          "hostPath": {
+            "path": "/var/vineyard/dump"
+          }
+        },
+        "pvc-spec": {
+          "storageClassName": "manual",
+          "accessModes": [
+            "ReadWriteOnce"
+          ],
+          "resources": {
+            "requests": {
+            "storage": "1Gi"
+            }
+          }
+        }
+        }'
```

Comment:
Could you please have some indentation for multiple-line commands? e.g.,

```bash
go run k8s/cmd/main.go -n backup create backup \
  --vineyard-name ... \
  -- ....
```

Rather than

```bash
go run k8s/cmd/main.go -n backup create backup \
--vineyard-name ... \
-- ....
```

to make them more readable...

Reply:
ok